notebooks/hello-npu/hello-npu.ipynb

```diff
@@ -0,0 +1,1249 @@
+{
```

Comment:
<p>could you please add --extra-index-url <a href="https://download.pytorch.org/whl/cpu" target="_blank">https://download.pytorch.org/whl/cpu</a></p><p> </p><p>this is needed for avoid downloading torch with cuda dependency (saves a lot of  disk space) if torch is not installed</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1898/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>